src/components/LargeDotGrid.js

```diff
@@ -0,0 +1,38 @@
+export default function LargeDotGrid({
+  width = 20,
+  height = 15,
+  space = 35,
+  radius = 4,
+  fill = "#A682FF",
+}) {
+  const viewWidth = width * radius * 2 + (width - 1) * (space - radius * 2);
```

Comment:
I wonder instead of having two separate components for the grids, is it possible to have one grid component listen for the screen size and apply different widths and heights conditionally.